resources/js/Pages/Dashboard/Course/Create.vue

```diff
@@ -75,6 +77,13 @@ function createCourse() {
               <TextInput id="semester" v-model="form.semester" type="number" min="1" max="10" :error="form.errors.semester" autocomplete="off" />
               <FormError :error="form.errors.semester" />
             </FormGroup>
+            <FormGroup>
```

Comment:
What about slug? Are you going to allow setting up custom ones?

Reply:
added